pkg/apiserver/api_installer.go

```diff
@@ -90,11 +91,11 @@ func (a *APIInstaller) Install(ws *restful.WebService) (apiResources []api.APIRe
 func (a *APIInstaller) NewWebService() *restful.WebService {
 	ws := new(restful.WebService)
 	ws.Path(a.prefix)
-	ws.Doc("API at " + a.prefix + " version " + a.group.Version)
+	ws.Doc("API at " + a.prefix + " version " + apiutil.GetGroupVersion(a.group.Group, a.group.Version))
```

Comment:
Wouldn't a user understand this a discrete piece of information? "API at $prefix, group $group, version $version"?